src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/perform_interaction_request.cc

```diff
@@ -153,14 +153,16 @@ void PerformInteractionRequest::Run() {
     }
   }
 
-  if (!CheckChoiceIDFromRequest(
-          app,
-          choice_set_id_list_length,
-          msg_params[strings::interaction_choice_set_id_list])) {
+  const auto result = CheckChoiceIDFromRequest(
+      app,
+      choice_set_id_list_length,
+      msg_params[strings::interaction_choice_set_id_list]);
+  if (mobile_apis::Result::SUCCESS != result) {
+    LOG4CXX_DEBUG(logger_, "HERE: " << result);
```

Comment:
@SKobziev , is this log still needed here? in case it is, please rephrase it to be more verbose

Reply:
This comment was removed because it was here only for a test. Could you please continue review?